lisa/sut_orchestrator/azure/transformers.py

```diff
@@ -165,10 +165,13 @@ def _prepare_virtual_machine(self, node: RemoteNode) -> None:
         if not runbook.public_address:
             runbook.public_address = node.public_address
 
-        # prepare vm for exporting
-        wa = node.tools[Waagent]
-        node.execute("export HISTSIZE=0", shell=True)
-        wa.deprovision()
+        try:
```

Comment:
Use a switch like `skip_wa_deprovision`. So it's applicable and restrict for different scenarios.